### PR TITLE
[Application] LLaMA v2

### PR DESCRIPTION
**Pull request by baek2sm**
Added LLaMA v2 application.

This implementations is based on llama of meta.
ref url: "https://github.com/facebookresearch/llama/"

It contains...
- implementations of swiglu, rmsprop, rotary embedding
- load weights from pytorch implementation(HuggingFace)

Signed-off-by: Seungbaek Hong <sb92.hong@samsung.com>

### CODE REVIEW

File: Applications/LLaMA/PyTorch/LLaMAv2.py [hunk 1]
```diff
@@ -0,0 +1,386 @@
+# License-Identifier: "LLAMA 2 COMMUNITY LICENSE"
+# License-Details: https://github.com/facebookresearch/llama/blob/main/LICENSE    
+# Copyright (C) 2023 Meta.
```
Comment (myungjoo) [inline Applications/LLaMA/PyTorch/LLaMAv2.py:3]: Are you sure that LLAMA2 community license is viral? (forcing this code as LLAMA2 community license)

Comment (baek2sm) [reply Applications/LLaMA/PyTorch/LLaMAv2.py:3]: @myungjoo Thank you for pointing out the important part! I will separate the file and follow the license only for the baseline code of the model, and I will write the rest of the additional part under the Apache-2.0 license. Thanks!

Comment (myungjoo) [reply Applications/LLaMA/PyTorch/LLaMAv2.py:3]: However, you may proceed with this PR and separate llama-licensed pard to another git repo later... as long as this file is not included in the product release (package file).

**Review by Hyoungsukim — APPROVED** [review]
lgtm

**Review by myungjoo — APPROVED** [review]

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**